### ISSUE TITLE
ERROR: Environment variable LD_LIBRARY_PATH not defined / Linux

### ISSUE THREAD

**Issue by chaytheeditors** [CoSMoSoftware/OBS-studio-webrtc#340]
I've just tried installing the latest version (m95, 2.25) on Zorin OS (Based on Ubuntu 20.04) and get the following error when trying to launch:
```
ERROR: Environment variable LD_LIBRARY_PATH not defined

Please set environment variable LD_LIBRARY_PATH to the path to the directory 'obs-plugins'

Usually directory 'obs-plugins' is to be found at /usr/lib

To set environment variable LD_LIBRARY_PATH to /usr/lib/obs-plugins, use the following command:
export LD_LIBRARY_PATH=/usr/lib/obs-plugins
```

I tried the command in terminal, but it doesn't recognise the command 'export'.

I was able to successfully install and run vanilla OBS, so doesn't seem to be an OBS / Zorin thing. 

I also tried installing the previous version (m95, 2.24) but got the same result.

**Comment by ludocosmo** [comment]
Command line `export LD_LIBRARY_PATH=/usr/lib/obs-plugins` is valid assuming your are using bash shell.
Depending on the shell you are using, you should modify the command line to add directory `/usr/lib/obs-plugins` to your environment variable `LD_LIBRARY_PATH`.

**Comment by sommerper** [comment]
I have the same problem. I installed `obs-webrtc-x64-27.1.3-m95-2.25-2004.deb` via Eddy on PopOS. Then I start the app via the Applications popup and the message appears.

I'm sorta new to Linux but I assume that putting `export LD_LIBRARY_PATH=/usr/lib/obs-plugins` in the shell only works in the shell.

Can I launch OBS from the shell somehow? I can't figure out where the binary was installed.

Thank you!

**Comment by chaytheeditors** [comment]
Yeah, I'm used to just launching this from the app icon (and usually on Windows or Mac OS). Is there a particular file that needs editing? `sudo bash` seems to launch into bash from Zorin terminal and I get no errors using `export LD_LIBRARY_PATH=/usr/lib/obs-plugins `, but then I still get the error when trying to launch.

**Comment by ludocosmo** [comment]
When you run command export `LD_LIBRARY_PATH=/usr/lib/obs-plugins` inside a bash shell, it sets the environment variable `LD_LIBRARY_PATH` for that shell only.
If you want the environment variable `LD_LIBRARY_PATH` to be set for your whole session, you need to set it in the local configuration file of your login shell.
* What is my login shell?
On a terminal, type:
`echo $SHELL`
* What is my local configuration file for my login shell?
It depends on your login shell.
For bash, local configuration files are `~/.bash_profile`, `~/.bash_login`, `~/.profile` and `~/.bashrc`
As your login shell is not bash, your local shell configuration file is not the same as for bash.
Make a search for your login shell to find what is its local configuration file and what is the syntax to set the environment variable `LD_LIBRARY_PATH` for that shell.
Add appropriate command lines for your login shell in your shell local configuration file to set `LD_LIBRARY_PATH`, then logout, and log in again. Open a shell to check that environemnt variable `LD_LIBRARY_PATH` has been set during login: `echo $LD_LIBRARY_PATH`

**Comment by ludocosmo** [comment]
> Can I launch OBS from the shell somehow? I can't figure out where the binary was installed.
> 
To launch OBS from the shell where you set `LD_LIBRARY_PATH`, simply type `obs`
Alternatively, you may type the whole path to the executable: `/usr/bin/obs`

**Comment by chaytheeditors** [comment]
Is this something that is likely to be fixed in a future release? As mentioned, expected behaviour would able to just launch this program from the dock/app list. There are also others in my company that will need to launch this and this will be far too much effort.
